src/sentry/tsdb/base.py

```diff
@@ -50,10 +50,20 @@ class TSDBModel(Enum):
 
     # distinct count of users that have been affected by an event in a group
     users_affected_by_group = 300
-
     # distinct count of users that have been affected by an event in a project
     users_affected_by_project = 301
 
+    # number of events sent to server for an organization (key is always 0)
+    frequent_organization_received_by_system = 400
+    # number of events rejected by server for an organization (key is always 0)
+    frequent_organization_rejected_by_system = 401
+    # number of events blacklisted by server for an organization (key is always 0)
+    frequent_organization_blacklisted_by_system = 402
```

Comment:
This is weird because there is an assumption in the API that all metrics have keys (in the case of the frequency table, a composite key) and I didn't want to overload the `internal` label with (`0`) another data type in case they end up having collisions.